src/backend/marsha/urls.py

```diff
@@ -38,6 +38,8 @@
     path(
         "lti/documents/<uuid:uuid>", DocumentLTIView.as_view(), name="document_lti_view"
     ),
+    # Public resources
+    path("video/<uuid:uuid>", VideoLTIView.as_view(), name="video_public"),
```

Comment:
It should get an "s" for coherence with the LTI view:
```suggestion
    path("videos/<uuid:uuid>", VideoLTIView.as_view(), name="video_public"),
```

Reply:
In fact I don't like this plural form.